features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -634,6 +634,10 @@
         Default value : false
         -->
         <!--<DropUnregisteredScopes>true</DropUnregisteredScopes>-->
+
+        <!-- Configuration for allowing users to introspect tokens from other tenants. -->
+        <AllowCrossTenantTokenIntrospection>true</AllowCrossTenantTokenIntrospection>
+
```

Comment:
Shall we remove this unwanted new line?

Reply:
Resolved by https://github.com/wso2/carbon-identity-framework/pull/4291/commits/2717d98a2f2acc6c2f361d7b59c657c9f3d1c5b7